nextcloud-startup-script.sh

```diff
@@ -155,18 +149,18 @@ debug_mode
 lowest_compatible_nc 16
 
 # Check that this run on the PostgreSQL VM
-if ! is_this_installed postgresql-10
+if ! is_this_installed postgresql-common
```

Comment:
is there no longer a specific version on 20.04?

Reply:
It's 12, but this is more generic.